app/controllers/catalog_controller.rb

```diff
@@ -114,6 +114,12 @@ class CatalogController < ApplicationController
     # common method since search results and reports all do the same configuration
     add_common_date_facet_fields_to_config! config
 
+    config.add_facet_field SolrDocument::FIELD_CONSTITUENTS, label: 'Virtual Objects', home: false,
+                                                             component: true,
+                                                             query: {
+                                                               has_constituents: { label: 'Has Constituents', fq: "#{SolrDocument::FIELD_CONSTITUENTS}:*" }
```

Comment:
I'm not sure if `Has Constituents` is the best label here. Defer to @andrewjbtw

Reply:
Maybe this is a theoretical question, but is it possible for something to tick the "I'm a virtual object" box and not have constituents? If so, having two values "has constituents" and "empty" could help with QA to identify empty objects. But if taking apart a virtual object removes it from the facet then I'd prefer the simplicity of "Virtual Object" - even if that ends up being redundant.

Reply:
I don't think that is possible here. This is a list of things in solr that have constituents. So there's a label for the facet header (currently `Virtual Objects`) and there's a label for the query (currently `Has Constituents`). Do you want one or both of these to change to `Virtual Object`?

Reply:
Ok, let's just go with "Virtual Objects" in both and not overthink it. I can already see a huge benefit in having the facet.

Reply:
@andrewjbtw OK, changed! Thank you.